app/components/modals/add-system-role-modal.js

```diff
@@ -0,0 +1,33 @@
+import FormMixin from 'open-event-frontend/mixins/form';
+import ModalBase from 'open-event-frontend/components/modals/modal-base';
+
+export default ModalBase.extend(FormMixin, {
+  isSmall            : true,
+  autoScrollToErrors : false,
+
+  actions: {
+    addRole() {
+      this.onValid(() => {
+        this.sendAction('addSystemRole');
+      });
+    }
+  },
+  getValidationRules() {
+    return {
+      inline : true,
+      delay  : false,
+      on     : 'blur',
+      fields : {
```

Comment:
@niranjan94 Will a multiple selection drop down be more suitable for this, instead of checkboxes 
https://semantic-ui.com/modules/dropdown.html
![img_3397](https://user-images.githubusercontent.com/17252805/27811077-8a2b95be-607f-11e7-973c-aab5859684ca.PNG)